wire-ios-data-model/Source/Proteus/ProteusError.swift

```diff
@@ -25,32 +25,32 @@ public enum ProteusError: Int, Error, Equatable {
     /// An error occurred while loading or peristing key material. The
     /// operation may be retried a limited number of times.
 
-    case storageError
+    case storageError = 1
```

Comment:
I noticed that all these error codes were off by one, because the original Cryptobox error enum started with a `success = 0` case.